console/console-init/ui/src/modules/iot-certificates/components/CertificateForm/CertificateForm.tsx

```diff
@@ -162,12 +164,12 @@ export const CertificateForm: React.FunctionComponent<ICertificateFormProps> = (
           <Grid>
             <GridItem span={6}>
               <FormGroup
-                fieldId={`cf-not-before-${id}`}
+                fieldId={`cert-form-not-before-text-${id}`}
                 isRequired
                 label={getLabelByKey("not-before")}
               >
                 <TextInput
-                  id={`cf-not-before-${id}`}
+                  id={`cert-form-not-before-text-${id}`}
```

Comment:
As stated earlier, this is an input field, and the id doesn't signify that. Either the id should have 'text-box' or 'input'.